src/plugins/timeConductor/Conductor.vue

```diff
@@ -458,6 +476,25 @@ export default {
             this.formattedBounds.end = this.timeFormatter.format(date);
             this.validateAllBounds('endDate');
             this.submitForm();
+        },
+        hideAllTimePopups() {
+            this.showTCInputStart = false;
+            this.showTCInputEnd = false;
+        },
+        showTimePopupStart() {
+            this.hideAllTimePopups();
+            this.showTCInputStart = !this.showTCInputStart;
+        },
+        showTimePopupEnd() {
+            this.hideAllTimePopups();
+            this.showTCInputEnd = !this.showTCInputEnd;
+        },
+        timePopUpdate(opts) {
+            let { type, hours, minutes, seconds } = opts;
```

Comment:
This can be done in the function signature to save a LOC, and make the opts behave like function arguments.

eg.
```
timePopUpdate({type, hours, minutes, seconds}) {
```

Reply:
> This can be done in the function signature to save a LOC, and make the opts behave like function arguments.
> 
> eg.
> 
> ```
> timePopUpdate({type, hours, minutes, seconds}) {
> ```

@jvigliotta This probably makes sense to you, can you address?